packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -225,6 +226,18 @@ module.exports = {
     ],
   },
   plugins: [
+    // Lint CSS with stylelint
+    new StyleLintPlugin({
+      // @remove-on-eject-begin
+      config: {
+        extends: 'stylelint-config-react-app',
+      },
+      configBasedir: paths.ownPath,
+      // @remove-on-eject-end
+      allowEmptyInput: true,
```

Comment:
Hm, right, this was merged into the `v8` branch. However version 8 is not yet out. The current version on npm is `7.10.1` however, so we cannot yet switch to this.